auth-api/Dockerfile

```diff
@@ -1,33 +1,75 @@
-FROM python:3.8.5-buster
+FROM python:3.8.16-bullseye as development_build
```

Comment:
Can you go to 3.9? As Python 3.8's end of life date, October 2024
eg
FROM python:3.9.19-bullseye AS development_build
eg. https://github.com/bcgov/lear/blob/main/queue_services/business-pay/Dockerfile

Reply:
Going to soon, just doing it incremental.. There's alot of changes for bumping python and that would be in the next PR or so.. We'd just push this out the way it is to PROD

Reply:
EG. SQLContinuum.. SQLAlchemy upgrade from 1.4 -> 2 etc.. alot of stuff to do for python upgrade

Reply:
> Can you go to 3.9? As Python 3.8's end of life date, October 2024 eg FROM python:3.9.19-bullseye AS development_build eg. https://github.com/bcgov/lear/blob/main/queue_services/business-pay/Dockerfile

i think next think will be pylint. as that will bring a bunch of changes. then follow it by other upgrades and all of those will lead up to python going to 3.12. I was originally trying to bring python up too but it was going to be a very massive change. hard to track where what went wrong.